contents/handbook/engineering/on-call-rotation.md

```diff
@@ -0,0 +1,39 @@
+---
+title: On-call rotation
+sidebar: Handbook
+showTitle: true
+---
+
+Every week, three engineers are on call in shifts. If you're one of the lucky three, congratulations!
+
+PostHog Cloud doesn't shut down at night (_whose_ night anyway?) nor on Sunday. As a 24/7 service, our goal is to be 100% operational 100% of the time. The universe doesn't always agree and hiccups do occur, which we catch in the form of service alerts. To make sure no alert goes unhandled, there is **always** one engineer on call, tasked with first-line defense.
+
+## When is my turn?
+
+We're very lucky that we have an almost-follow-the-sun rotation for service alerts, so nobody gets woken up in the middle of the night. Every engineer at PostHog is expected to be part of the rotation within one of the three PagerDuty schedules:
+
+- [Europe](https://posthog.pagerduty.com/schedules#PF7ZGBT) (08:00 to 16:00 UTC)
+- [Americas East](https://posthog.pagerduty.com/schedules#PW1E9Y4) (16:00 to 00:00 UTC)
+- [Americas West](https://posthog.pagerduty.com/schedules#P3J10CZ) (00:00 to 08:00 UTC)
+
+Before going on call, make sure you have the [PagerDuty mobile app](https://support.pagerduty.com/docs/mobile-app) installed and configured. This way it'll be harder to miss an alert.
+
```

Comment:
```suggestion
Read and make sure you have the necessary access here: http://runbooks/oncall
```

Reply:
Good point about access, but added it to the "What do I do on call?" section where runbooks are already explained

Reply:
this is a specific runbook page folks should review before they are oncall - so it falls under "Before going on call" as you mention here. We don't want them to start sorting out access after getting paged

Reply:
Oh, right. But it covers the same thing as this very section this PR is about. IMO this is exactly the stuff that should live in the handbook, which is the thing to read _before_ going on call. Then http://runbooks can be the streamlined resource for handling an alert, without higher-level distractions like this (though it can definitely link here).

I inlined the access section into this doc, let me know if it's good to go. [`91157a7` (#7432)](https://github.com/PostHog/posthog.com/pull/7432/commits/91157a7b54fd26d07cf42645442f84fa83bb9f69)